fuzzing/seeds/rules/app-layer-events.rules

```diff
@@ -0,0 +1,18 @@
+# App layer event  rules
```

Comment:
Why did you copy this file ?

Reply:
I test the rule parser and need this files as an seed-input

Reply:
I use emerging threats to have a more complete corpus : cf https://github.com/google/oss-fuzz/blob/fe897ed8ced473f690be320c8087e74e9ab62761/projects/suricata/build.sh#L60

Reply:
I'll try is with my fuzz-targets